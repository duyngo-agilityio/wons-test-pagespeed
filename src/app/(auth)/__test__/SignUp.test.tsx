import { render, waitFor } from '@testing-library/react';

// Pages
import SignUpPage from '@/app/(auth)/sign-up/page';

describe('SignUp Page render', () => {
  it('should render and match with snapshot', async () => {
    const { container } = render(<SignUpPage />);

    await waitFor(() => {
      expect(container).toBeInTheDocument();
    });

    expect(container).toMatchSnapshot();
  });
});
