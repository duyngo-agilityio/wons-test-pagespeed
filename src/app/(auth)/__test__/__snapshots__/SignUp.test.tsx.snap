// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignUp Page render should render and match with snapshot 1`] = `
<div>
  <div
    data-testid="sign-up-form"
  >
    Sign Up Form
  </div>
</div>
`;
